src/pages/test-report/cat-cpc/components/question-footer/question-footer.ts

```diff
@@ -32,7 +46,36 @@ export class QuestionFooterComponent {
   }
 
   goToSummary = (): void => {
-    this.testSummaryRequested.emit(true);
+    if (this.isDelegated) {
+      if (this.isFormValid()) {
+        this.testSummaryRequested.emit(true);
+      }
```

Comment:
the function could `return` here and there is no need for the `else`

Reply:
Added return at the end of the isDelegated if and removed the else